Task_1.py

```diff
@@ -1,160 +1,165 @@
-                break
+    '''function returns the application which received the most useless message
+
+    1.3. Task_1.py: to create file general-stats.cvs containing information about
+    the application which received the most useless message;
+    '''
+    bad_comment = min(
+        dict(filter(lambda review: review[1]['helpful'][1] != 0, main_dict.items())).values(),
+        key=lambda review: review['helpful'][0]/review['helpful'][1]
+    )
+    comment = [
+        ['The application which received the most useless message:'],
+        ['helpfulness:', str(bad_comment['helpful'][0]/bad_comment['helpful'][1]*100) + '%'],
+        ['asin:', bad_comment['asin']],
+        ['reviewText:', bad_comment['reviewText']]
+    ]
     return comment
 
-# 1) Task_1.py: to create file general-stats.cvs containing information about:
-#    1.5. the number of records that cannot be processed for every point above.
 def nonanalys_data(main_dict, analyzed_data):
-    comment = []
+    '''function returns the number of records that cannot be processed for every point above.
+
+    1.5. Task_1.py: to create file general-stats.cvs containing information about
+    the number of records that cannot be processed for every point above.
+    '''
+    comment = [['The number of records that cannot be processed: ']]
+
     count_unanalyzed_1 = 0
     for item in main_dict.values():
         if not 'asin' in item.keys() or not 'overall' in item.keys():
             count_unanalyzed_1 += 1
-    comment_1 = [
-        str(count_unanalyzed_1) + ' - the number of records that cannot be processed '
-                                'for average rating (overall) of each application (asin)']
-    comment.append(comment_1)
+    comment.append([count_unanalyzed_1, ' - for average rating (overall) of each application (asin)'])
 
     count_unanalyzed_2 = 0
     for item in main_dict.values():
         if not 'asin' in item.keys() or not 'helpful' in item.keys() or not 'reviewText' in item.keys():
             count_unanalyzed_2 += 1
-    comment_2 = [
-        str(count_unanalyzed_2) + ' - the number of records that cannot be processed '
-                                'to get the application which received the most useless message']
-    comment.append(comment_2)
-
-    count_unanalyzed_3 = len(main_dict) - len(set(analyzed_data))
-    comment_3 = [
-        str(count_unanalyzed_3) + ' or ' + str(round(count_unanalyzed_3/len(main_dict) * 100)) + '% ' +
-        ' - the number of records that cannot be processed to get the shortest interval between '
-        'ratings of one user (among all users) and the length of both messages which create this interval;']
-    comment.append(comment_3)
+    comment.append([count_unanalyzed_2, ' - to get the application which received the most useless message'])
+
+    count_unanalyzed_3 = len(main_dict) - analyzed_data
+    comment.append([str(count_unanalyzed_3) + ' or ' + str(round(count_unanalyzed_3/len(main_dict) * 100)) + '% ',
```

Comment:
Please, use [string interpolation](https://www.python.org/dev/peps/pep-0498/), it's much more convenient

Reply:
I see. Will do!